tasks/setup_environment.yml

```diff
@@ -44,6 +44,3 @@
 - name: Confirm deployment details
   ansible.builtin.pause:
     prompt: Press 'Enter' to continue with the deployment or Ctrl+C to cancel
-
-- name: Include configure ufw task
```

Comment:
any reason to move it out of here to node task file ? I think its better suited here